src/components/Molecules/SchoolLookup/SchoolLookup.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import 'jest-styled-components';
+
+import renderWithTheme from '../../../hoc/shallowWithTheme';
+import SchoolLookup from './SchoolLookup';
+
+// todo: can we test the interactions?
```

Comment:
Usually we test interactions in our app. We can add some cypress tests in the integration part.